### PR TITLE
release: v0.1.4

### PR DESCRIPTION
**Pull request by jahkeup**
**Description of changes:**

Bumps release data for `v0.1.4`:

- `CHANGELOG.md`
- `VERSION`
- Kubernetes deployments

**Testing done:**

- functional test ~~in progress~~ are passing!
- Cluster of 20 nodes upgrades and stabilizes
  - Used `aarch64` and `x86_64` in a mixed cluster (multi-arch images)

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by etungsten — APPROVED** [review]

**Review by samuelkarp — APPROVED** [review]

**Comment by jahkeup** [comment]
Blocked on #39, will rebase to include #39 when approved.

**Comment by jahkeup** [comment]
Updated with fixes and last minute changes. Changelog updated to reflect these as well.

File: update-operator.yaml [hunk 1]
```diff
@@ -118,7 +119,7 @@ spec:
       containers:
       - name: controller
         # TODO Update this field to the new version that supports update API once that's released
```
Comment (etungsten) [inline update-operator.yaml]:
```suggestion
```

File: update-operator.yaml [hunk 2]
```diff
@@ -166,11 +167,12 @@ spec:
                     operator: In
                     values:
                       - amd64
+                      - arm64
       hostPID: true
       containers:
         - name: agent
           # TODO Update this field to the new version that supports update API once that's released
```
Comment (etungsten) [inline update-operator.yaml]:
```suggestion
```

**Review by etungsten — APPROVED** [review]
small nits

File: update-operator.yaml (hunk 1 above)
Comment (jahkeup) [reply update-operator.yaml]: Oh, nice catch! Will update.

**Review by webern — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**